tools/decode_jwt_payload.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
```

Comment:
I'm not sure of the "why" of this python script.

Reply:
The goal for this python script is to support the use case of "upload a profile then gets the hash", in tests for example. We've seen it was advocated in some tests in the past. That's why I provide this python script, where we'd just need to pipe the output of the request to get the hash back.
I want to write some more detailed documentation later where I'd explain how one can use the API.